core/src/main/java/com/graphhopper/routing/util/parsers/CarAccessParser.java

```diff
@@ -93,6 +93,14 @@ public WayAccess getAccess(ReaderWay way) {
             return WayAccess.CAN_SKIP;
         }
 
+        if ("pedestrian".equals(highwayValue)) {
+            // allow access if explicitly tagged
+            String restriction = way.getFirstValue(restrictionKeys);
+            if (!intendedValues.contains(restriction)) {
```

Comment:
By using `intendedValues` we can skip any tagging mistakes like `motor_vehicle=asdf`